Makefile

```diff
@@ -18,12 +18,12 @@ OUTPUT ?= $(JAVA_DIR)
 LANGUAGE ?= java
 GRPCPLUGIN ?= /usr/bin/protoc-gen-grpc-$(LANGUAGE)
 # with https://github.com/TheThingsIndustries/docker-protobuf/issues/9 we could use this image also for TypeScript
-PROTOC ?= docker run --rm -v $(DIR):$(DIR) -w $(DIR) thethingsindustries/protoc
+PROTOC ?= docker run --rm -v $(DIR):$(DIR) -w $(DIR) protoc-harry
 
-FLAGS+= --proto_path=.
+FLAGS+= --proto_path=$(DIR)
 ifeq ($(LANGUAGE),node)
-FLAGS+= --plugin=protoc-gen-ts=$(NODE_DIR)/node_modules/.bin/protoc-gen-ts
-FLAGS+= --plugin=protoc-gen-grpc=$(NODE_DIR)/node_modules/.bin/grpc_tools_node_protoc_plugin
+FLAGS+= --plugin=protoc-gen-ts=/node_modules/.bin/protoc-gen-ts
```

Comment:
paths are absolut cause we know the location in the image

Reply:
`FLAGS+= --grpc_out=$(OUTPUT)` should be probably be `FLAGS+= --grpc_out=$(NODE_DIR)`